### DISCUSSION TITLE
How to use the lib in a Qt widget ?

### DISCUSSION THREAD

**Discussion by virtuamechanica** [alandefreitas/matplotplusplus#402] [Q&A]
Hello
I would like to embedd a contourplot in a Qt dialog (as widget) in C++, as an alternative to QtCustom mPlot lib (which has no MIT license).  Faisible ?
Thank you!

**Comment by IhabRagabMansour** [comment]
Did you find an answer?

**Comment by alandefreitas** [comment]
In practice, no. See https://github.com/alandefreitas/matplotplusplus/discussions/445
